### ISSUE TITLE
Add BlogPosting properties

### ISSUE THREAD

**Issue by dominhhai** [kakawait/hugo-tranquilpeak-theme#209]
[Google Web Structured Data Testing Tool](https://search.google.com/structured-data/testing-tool)
output some error about missing [BlogPosting](https://schema.org/BlogPosting) properties as
`author`, `image`, `publisher` and warming missing as `dateModified`, `mainEntityOfPage`.
So, should we add these properties for blog post?
It is nice if them can be configured for each post.

**Comment by kakawait** [comment]
@dominhhai thank for reporting I will firstly check to add it (hardcode) and then add a way to custom it

**Comment by kakawait** [comment]
@dominhhai I post-poned that issue for 3 reasons:

1. I have to teach myself about schema.org standard
2. I have to rethink the content to insert `author`, `publisher` data
3. I will not hardcode the value, instead (thank to post-pone) I will do what you propose a way to change value for each post

**Comment by dominhhai** [comment]
This maybe good start point for front-end developers.
https://github.com/thedaviddias/Front-End-Checklist

**Comment by kakawait** [comment]
@dominhhai Yes I saw that checklist on _Github trending_ I have to read it, because even if I'm author of that blog I'm not frontend developer, I'm backend developer that why I lack some knowledge

**Comment by dominhhai** [comment]
Your front-end code is pretty 💯

**Comment by kakawait** [comment]
@dominhhai created a meta issue https://github.com/kakawait/hugo-tranquilpeak-theme/issues/222 (I will fill it asap, I started but no read all document)

**Comment by dominhhai** [comment]
As you see, or my PR, I also add some suggestion tag from Front-End-CheckList.

**Comment by kakawait** [comment]
Yes i saw that. I will handle the PR for 0.4.3-BETA since i will focus on opengraph for that release.
